### DISCUSSION TITLE
7.63.0 -> 7.88.1, when TLS 1.2 idle connection closed by server

### DISCUSSION THREAD

**Discussion by RabidRabbit2019** [curl/curl#11522] [Q&A] [answered]
7.63.1: when TLS 1.2 idle connection closed by server, at next curl_easy_perform() server send RST and libcurl create new connection
server: Encrypted Alert
client: ACK
server: FIN, ACK
client: ACK
after 450 seconds:
client: Application Data
client: Encrypted Alert
server: RST
client: SYN
server: SYN, ACK
etc.

7.88.1: when TLS 1.2 idle connection closed by server, at next curl_easy_perform() server send RST and libcurl return error 55
so, we rolled back to 7.63.0
flow same as above, but after RST from server libcurl return error 55.

q: how make libcurl 7.88.1 reconnect after write into closed connection?

**Comment by bagder** [comment, marked as answer]
That sounds like a bug. I would suggest you upgrade and if the problem persists, file an issue.
